src/tasks/installers/operator.ts

```diff
@@ -361,16 +361,34 @@ export class OperatorTasks {
     {
       title: `Delete the Custom Resource of type ${CHE_CLUSTER_CRD}`,
       task: async (_ctx: any, task: any) => {
+        await kh.deleteCheCluster(flags.chenamespace)
+        do {
+          await cli.wait(20 * 60 * 1000) // default timeout in che operator to clean up cluster scope objects
+        } while (await kh.getCheCluster(flags.chenamespace))
```

Comment:
Sorry, I'm afraid chectl could hangs here...

Reply:
Tested. With a fake finalizer.
checluster is removed when finalizers are removed by patching CR
```
await kh.patchCheClusterCustomResource(checluster.metadata.name, flags.chenamespace, { metadata: { finalizers: null } })
```